### PR TITLE
ppu/spu: improve accuray of some instructions

### PR DESCRIPTION
**Pull request by scribam**
Tests from https://github.com/RPCS3/ps3autotests/

**fnmsub (ppu)**

|   | Success  | Total  | Accuracy  |  
|---|---|---|---|
| Master LLVM  | 8549 | 10648 | 80.29%  |  
| PR LLVM  | 10301 | 10648 | 96.74% | 
| Master Fast  | 8897 | 10648 | 83.56%  |  
| PR Fast | 10648 | 10648 | 100% | 

**fnmsubs (ppu)**

|   | Success  | Total  | Accuracy  |  
|---|---|---|---|
| Master LLVM  | 8893 | 10648 | 83.52%  |  
| PR LLVM  | 10646 | 10648 | 99.98% | 
| Master Fast  | 8893 | 10648 | 83.52%  |  
| PR Fast | 10646 | 10648 | 99.98% | 

**vnmsubfp (ppu)**

|   | Success  | Total  | Accuracy  |  
|---|---|---|---|
| Master LLVM  | 2857 | 10648 | 26.83%  |  
| PR LLVM  | 10354 | 10648 | 97.24% | 
| Master Fast  | 2983 | 10648 | 28.01%  |  
| PR Fast | 10641 | 10648 | 99.93% | 

**dfnma (spu)**

|   | Success  | Total  | Accuracy  |  
|---|---|---|---|
| Master LLVM  | 3781 | 10648 | 35.51%  |  
| PR LLVM  | 4554 | 10648 | 42.77%  | 
| Master Asmjit  | 3115 | 10648 | 29.25%  |  
| PR Asmjit | 4554 | 10648 | 42.77% | 
| Master Fast  | 3221 | 10648 | 30.25%  |  
| PR Fast | 4653 | 10648 | 43.70% |

### CODE REVIEW

**Comment by Nekotekina** [comment]
In case you didn't see

https://cdn.discordapp.com/attachments/303330503392755712/461820110542340136/unknown.png

**Comment by woj1993** [comment]
@scribam Can you give us dfnma accuracy before and after?

**Comment by scribam** [comment]
Updated

**Comment by TGEnigma** [comment]
Mind rebasing?

File: rpcs3/Emu/Cell/SPUInterpreter.cpp [hunk 1]
```diff
@@ -1090,7 +1090,7 @@ bool spu_interpreter_fast::DFNMS(SPUThread& spu, spu_opcode_t op)
 
 bool spu_interpreter_fast::DFNMA(SPUThread& spu, spu_opcode_t op)
 {
-	spu.gpr[op.rt].vd = _mm_sub_pd(_mm_set1_pd(0.0), _mm_add_pd(_mm_mul_pd(spu.gpr[op.ra].vd, spu.gpr[op.rb].vd), spu.gpr[op.rt].vd));
+	spu.gpr[op.rt].vd = _mm_xor_pd(_mm_add_pd(_mm_mul_pd(spu.gpr[op.ra].vd, spu.gpr[op.rb].vd), spu.gpr[op.rt].vd), _mm_set1_pd(-0.0f));
```
[the remark this answers is not in the document]
Comment (scribam) [reply rpcs3/Emu/Cell/SPUInterpreter.cpp]: Indeed. Fixed, thanks for the review.

**Pull request merged**